README.md

```diff
@@ -181,8 +181,8 @@ In a near feature we will support:
 
 - fix number of labels for discrete values (delegate method)
 - details on demand
-- ~~~2D/3D graph visualisation (node-link diagrams) directly in the *Visualisation* editor~~~
-- more type of data sources (including real time data)
+- 2D/3D graph visualisation (node-link diagrams) directly in the *Visualisation* editor
+- ~~more type of data sources (including real time data)~~
```

Comment:
Note to self, alter to only have the "including real time data" to be strike through